README.md

```diff
@@ -44,6 +46,9 @@ See below for using the build script.
 
 * You can also run barys with the -h switch to inspect the available options
 
+* You can also build via Docker to ensure you have all the right dependencies. E.g. via
+  `docker run --rm -it --privileged -v $(pwd):/home/build aggurio/docker-balena  ./balena-yocto-scripts/build/barys -k -d -m pine64-plus`
```

Comment:
We don't advertise this as it's not an official product that we want to support. We are working on some changes in meta-balena that will build in containers by default.

Reply:
all right, will remove this with my next commits